### ISSUE TITLE
RED ALERT! HALP! ACTUALLY!

### ISSUE THREAD

**Issue by LostKing73** [cs6413110/Pixel-Tanks#176]
@cs6413110 HALP! FLAT MADE KEEGAN ADMIN
![image](https://github.com/user-attachments/assets/12e8a846-a324-4365-bd68-6fb9d707366d)

**Comment by LostKing73** [comment]
NOT GOOD

**Comment by LostKing73** [comment]
REMOVE

**Comment by LostKing73** [comment]
![image](https://github.com/user-attachments/assets/de2906f3-8087-4f12-96b3-5d9f9303b31c)

**Comment by DarkMemeGod411** [comment]
So keegan gets admin? but lk doesnt?

**Comment by DarkMemeGod411** [comment]
> rip gam e for a week

wdym

**Comment by BreadSpammer** [comment]
nothing we can do about it

**Comment by DarkMemeGod411** [comment]
why?

**Comment by BreadSpammer** [comment]
bc black mail

**Comment by DarkMemeGod411** [comment]
So, we cant remove his admin?

**Comment by cs6413110** [comment]
Nah, let game burn for a few days. This is prime reasoning for removing flat.

Also:
![Screenshot 2024-11-18 10 05 15 AM](https://github.com/user-attachments/assets/27c0e97a-def5-46d2-b629-c44b45de3c18)

**Comment by LostKing73** [comment]
like figs :(

**Comment by BreadSpammer** [comment]
nah

**Comment by 3foe** [comment]
bruh keegan stole my Chromebook while I had to do a hearkness discussion in 2nd hour
ill fix now

**Comment by LostKing73** [comment]
> bruh keegan stole my Chromebook while I had to do a hearkness discussion in 2nd hour ill fix now

uh huh. sure.

**Comment by 3foe** [comment]
I was logged in and goingto fight keegan cause he said he could beat me but Mr treat me up to debate jace in the hearkness. (I won BTW by about 20 points) during that time he yoinked my CB but I couldn't do anything bout it cause I was debating jace.

**Comment by 3foe** [comment]
i can't log in to fix it bread broke sign in

**Comment by LostKing73** [comment]
> I was logged in and goingto fight keegan cause he said he could beat me but Mr treat me up to debate jace in the hearkness. (I won BTW by about 20 points) during that time he yoinked my CB but I couldn't do anything bout it cause I was debating jace.

uh huh. sure.

**Comment by LostKing73** [comment]
> i can't log in to fix it bread broke sign in

wasn't bread

**Comment by LostKing73** [comment]
so... just gonna put this out there... flat made keegan admin. keegan didn't make keegan admin

**Comment by 3foe** [comment]
dude, I dident, why would I make keegan admin?

**Comment by 3foenation** [comment]
> > bruh keegan stole my Chromebook while I had to do a hearkness discussion in 2nd hour ill fix now
> 
> uh huh. sure.

he did say he gave it to himself

**Comment by LostKing73** [comment]
> dude, I dident, why would I make keegan admin?

because keegan isn't a liar (unlike you) and he told me he didn't give to himself

**Comment by LostKing73** [comment]
> > > bruh keegan stole my Chromebook while I had to do a hearkness discussion in 2nd hour ill fix now
> > 
> > 
> > uh huh. sure.
> 
> he did say he gave it to himself

he didn't. flat said he gave it to himself
